### DISCUSSION TITLE
Problems running Copilot in VS Code with multiple github accounts

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18684] [Copilot Conversations] [answered]
I have a github personal account as well as a corporate account. I've been approved for access using my corporate account, but when I tried to activate it there was some kind of problem, and now I can't bring up the "sign into github" panel any more, even if I uninstall and then reinstall the plugin. In fact, it seems to have forgotten my approval state, since now I see an error saying 

```
[INFO] [auth] [2022-06-15T17:01:45.301Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-06-15T17:01:45.303Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```
How do I tell which acccount it was trying to log in with? How do I bring up the login panel again?

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by GoldbergGLG** [comment]
My issue is, I'd like to avoid paying twice, at least until the enterprise version comes out.  I am already paying for my personal.  I need to keep both accounts.  Thoughts?
